recipes/CheckmarxPythonSDK/LICENSE.txt

```diff
@@ -0,0 +1,674 @@
+                      GNU GENERAL PUBLIC LICENSE
```

Comment:
The license is in the package tarball, no need to add it here.

Reply:
It is done, thanks. Previously I got an error related to the licenses that I miss understood. I now see that was because I added a .txt extension, that in this case is not needed